src/modules/profile/views/waitingTimeStatistics/waitingTimeStatistics.tsx

```diff
@@ -1,14 +1,12 @@
-import { ArcElement, Chart as ChartJS, type ChartOptions, Legend, Tooltip } from 'chart.js';
-import { useEffect, useMemo, useState } from 'react';
-import { Pie } from 'react-chartjs-2';
-
-ChartJS.register(ArcElement, Tooltip, Legend);
+import { useMemo } from 'react';
+import WaitingTimeChart from './waitingTimeChart';
+import { sortBy } from 'lodash';
```

Comment:
Please use this way to import.
`import sortBy from 'lodash/sortby'`

**Below shows the difference in _bundle size_**

![image](https://github.com/paziresh24/patient/assets/46134157/391938f7-93d7-4a88-aba6-f664f342a7c5)
![image](https://github.com/paziresh24/patient/assets/46134157/59ec1cc6-3183-4686-a1dc-0ac151dcde2b)